README.org

```diff
@@ -59,10 +59,23 @@ capabilities]].
 
 Installing to the homescreen on:
 
-- iOS: Open organice in Mobile Safari. Tap the "share" button and
+- **iOS:** Open organice in Mobile Safari. Tap the "share" button and
```

Comment:
Here's a conflict to the PR that was just opened before yours: https://github.com/200ok-ch/organice/pull/302

Reply:
OK, fixing.